core/src/tasks/record/export.ts

```diff
@@ -19,59 +20,44 @@ export class RecordExport extends RetryableTask {
   };
 
   async runWithinTransaction({ force, recordId }: ParamsFrom<RecordExport>) {
-    const record = await GrouparooRecord.findOne({ where: { id: recordId } });
+    const record = await GrouparooRecord.findOne({
+      where: { id: recordId },
+      include: [RecordProperty, { model: GroupMember, include: [Group] }],
+    });
 
     if (!record) return; // the record may have been deleted or merged by the time this task ran
     if (record.state !== "ready") return;
 
     const imports = await Import.findAll({
       where: {
-        state: "processing",
+        state: "complete",
         recordId: record.id,
       },
       order: [["createdAt", "asc"]],
     });
```

Comment:
Previously we had only been looking up imports that had not created exports, whereas now we are looking up all complete imports. I _believe_ this is only being used to check which specific destinations we need to export to, but it would be including all destinations we've ever wanted to export to instead of the ones from the latest imports.

Reply:
That's a really good point - maybe the `processing` state needs to remain for the imports for this reason